Source/Scene/Model.js

```diff
@@ -1077,8 +1079,35 @@ define([
                             var position = primitives[m].attributes.POSITION;
                             if (defined(position)) {
                                 var accessor = gltfAccessors[position];
-                                var aMin = Cartesian3.fromArray(accessor.min, 0, aMinScratch);
-                                var aMax = Cartesian3.fromArray(accessor.max, 0, aMaxScratch);
+                                var extensions = accessor.extensions;
+                                var accessorMin = new Matrix(accessor.min);
+                                var accessorMax = new Matrix(accessor.max);
+                                
+                                // If this accessor is quantized, we should use the decoded min and max
+                                if (defined(extensions)) {
+                                    var quantizedAttributes = extensions.WEB3D_quantized_attributes;
+                                    if (defined(quantizedAttributes)) {
```

Comment:
> However, I think an appropriate solution might be to just make decodeMin and decodeMax non-optional with POSITION attributes like we proposed doing with the regular accessor.min and max

Yes, I agree, let's propose this to @mlimper and offer and pull request to update the extension.

> That being said, it probably isn't a bad idea to have a more general way to do matrix math in Cesium...

Yes, but we would not merge it into master or put more effort into developing it until it is definitively needed; otherwise, it is extra code that needs to be deployed and maintained.

If any extension spec decisions are holding this PR up, let's turn our attention to the gltf-pipeline implementation.